dev-util/idea-ultimate/idea-ultimate-15.0.2.143.1015.7_pre.ebuild

```diff
@@ -73,7 +73,9 @@ src_install() {
 	doins -r *
 	fperms 755 "${dir}"/bin/{idea.sh,fsnotifier{,64}}
 
-	make_wrapper "${PN}" "${dir}/bin/${MY_PN}.sh"
+	make_wrapper "${PN}" "${dir}/bin/${MY_PN}.sh" || die
+	newicon "bin/${MY_PN}.png" "${PN}.png" || die
+	make_desktop_entry "${PN}" "IntelliJ Idea Ultimate" "${PN}" "Development;IDE;" || die
```

Comment:
These rules are there since years. Could you please point me where you have been corrected the other way round?